Automattic/jetpack
# VideoPress: Paginate and search while uploading

Fixes #26701 

#### Changes proposed in this Pull Request:

* Do not disable search, filter and pagination while uploading.
* Set correct key for items in `VideoGrid` to avoid re-render problems.
* Fill loading mode with unique id, to re-render correctly the list.
* Mark as fullfiled if `id` is string in `getVideo` resolver.
* Do not show `pagination` if total videos is equal `totalItems`.
* Invalidate `uploadVideoCount` in specific actions.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to `Dashboard`.
* Make sure you have other videos, and start to upload.
* Search or paginate, it should work normally while uploading.

#### Demo

https://user-images.githubusercontent.com/1663717/195214679-9307da9a-2128-457c-b1a9-3853d5ac4590.mp4
